news-bundle/src/Resources/contao/languages/de/tl_news.xlf

```diff
@@ -327,4 +327,4 @@
       </trans-unit>
     </body>
   </file>
-</xliff>
```

Comment:
This file does not need to be adjusted :) (you've deleted the final new line. Our coding style forces to always end a file with a new line :)